src/controllers/SocialsController.js

```diff
@@ -156,19 +161,20 @@ class SocialController {
       id, username, email, signUpType,
     };
     const token = TokenAuthenticate.generateToken(payload, tokenExpireTime);
+
     if (newUser) {
       response = new Response(
         'Ok',
         201,
         'User logged in successfully',
         { token }
       );
-      return res.status(response.code).json(response);
+      return res.redirect(`${FRONTEND_BASE_PATH}/auth?token=${token}`);
```

Comment:
Why are you redirecting from the server? I'll go ahead and merge this in the meantime though